### PR TITLE
CHG5015777 - shuttering Thursday 18th April DO NOT MERGE

### PR DESCRIPTION
**Pull request by RebeccaBaker**
### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: shuttering/prod/apply-divorce-service-gov-uk.yml [hunk 1]
```diff
@@ -19,18 +19,18 @@ A: []
 
 cname:
   - name: "*"
-    shutter: false
+    shutter: true
```
Comment (ieuanb74) [inline shuttering/prod/apply-divorce-service-gov-uk.yml:22]: Can we change this back to false, "*" records don't currently work with the shuttering process. The service will still be shuttered by leaving the rest of the records as true

Comment (RebeccaBaker) [reply shuttering/prod/apply-divorce-service-gov-uk.yml]: done 👍

File: shuttering/prod/apply-to-adopt-a-child-placed-in-your-care-service-gov-uk.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ shutter_all: false
 # - alias_target_resource_id
 shutter_all_a: false
 
-shutter_all_cname: false
+shutter_all_cname: true
```
Comment (ieuanb74) [inline shuttering/prod/apply-to-adopt-a-child-placed-in-your-care-service-gov-uk.yml:12]: This dns zone uses A records for shuttering so we need shutter_all_a set to true and shutter_all_cname set to false as not required

File: shuttering/prod/end-civil-partnership-service-gov-uk.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@ shutter_all: false
 # - alias_target_resource_id
 shutter_all_a: false
 
-shutter_all_cname: false
+shutter_all_cname: true
```
Comment (ieuanb74) [inline shuttering/prod/end-civil-partnership-service-gov-uk.yml:12]: As this one includes a "*" records we need to set this to false. Can enable shuttering by setting the individual "www" record to true

File: shuttering/prod/hmcts-access-service-gov-uk.yml [hunk 4]
```diff
@@ -9,7 +9,7 @@ shutter_all: false
 # - alias_target_resource_id
 shutter_all_a: false
 
-shutter_all_cname: false
+shutter_all_cname: true
```
Comment (ieuanb74) [inline shuttering/prod/hmcts-access-service-gov-uk.yml:12]: This one also shutters with A records. Need shutter_all_a set to true and shutter_all_cname set to false

File: shuttering/prod/apply-to-adopt-a-child-placed-in-your-care-service-gov-uk.yml (hunk 2 above)
Comment (RebeccaBaker) [reply shuttering/prod/apply-to-adopt-a-child-placed-in-your-care-service-gov-uk.yml]: done 👍

File: shuttering/prod/end-civil-partnership-service-gov-uk.yml (hunk 3 above)
Comment (RebeccaBaker) [reply shuttering/prod/end-civil-partnership-service-gov-uk.yml]: done

File: shuttering/prod/hmcts-access-service-gov-uk.yml (hunk 4 above)
Comment (RebeccaBaker) [reply shuttering/prod/hmcts-access-service-gov-uk.yml]: done

**Review by ieuanb74 — APPROVED** [review]

**Pull request merged**